package.json

```diff
@@ -38,12 +38,19 @@
     "identity",
     "verification"
   ],
-  "files": ["dist", "lib", "src"],
+  "files": [
+    "dist",
+    "lib",
+    "src"
+  ],
```

Comment:
In the previous PR, this array was changed to be inline, now it's back as it was. Have we changed the linter rule?

Reply:
Not sure why, maybe because of the TypeScript parser 🤔